.github/workflows/create-release-pull-request.yaml

```diff
@@ -21,10 +21,19 @@ jobs:
 
       - name: Start Changelog entry
         run: |
-          function dt {
-            date +'%Y-%m-%d'
-          }
-          sed -i "s/<!--Start Of Releases (DO NOT DELETE THIS LINE)-->/<!--Start Of Releases (DO NOT DELETE THIS LINE)-->\n\n## [${{ github.event.inputs.newVersionNumber }}] - $(date +%F)\n\n## Highlights\n\n## What’s Changed\n\n## New Contributors/" ./CHANGELOG.md
```

Comment:
Could we still add a blank `Highlights` and `New Contributors` section still? Seem to have lost these with this PR.

Reply:
https://github.com/sqlfluff/sqlfluff/pull/3360/commits/5df1f207949b8d9f7ac0b0996ec60ebbef5cb5d6

Resulted in: https://github.com/greg-finley/sqlfluff/pull/28